### PR TITLE
Add village and other fields based on kobo form changes

### PR DESCRIPTION
**Pull request by wadhwamatic**
Add new village field to the three forms

Update Incident report to add a new shipwreck incident type and factory counts
Update Flood report to add 3 new fields

Update Readme to guide users through column additions

### CODE REVIEW

File: packages/interfaces/src/kobo/IncidentDto.ts [hunk 1]
```diff
@@ -86,6 +86,8 @@ export class IncidentDto {
   @IsOptional() @IsNumberString() readonly 'group_pk03i04/group_ub2hp14/CraftDam'?: string;
   @IsOptional() @IsNumberString() readonly 'group_gh6ag70/group_na4yi04/CropAff'?: string;
   @IsOptional() @IsNumberString() readonly 'group_gh6ag70/group_na4yi04/CropDam'?: string;
+  @IsOptional() @IsNumberString() readonly 'group_gh6ag70/group_az7bv13/integer_ws0la67'?: string;
```
Comment (ericboucher) [inline packages/interfaces/src/kobo/IncidentDto.ts:89]: @wadhwamatic I think they should update these names before they get integrated in other apps to stay consistent with the rest

Comment (wadhwamatic) [reply packages/interfaces/src/kobo/IncidentDto.ts:89]: agreed. I'll have them update it in kobo

File: apps/frontend/translations/en.json [hunk 2]
```diff
@@ -211,8 +211,13 @@
         "NumWareHDam": "Damaged",
         "NumCraftAff": "Affected",
         "NumCraftDam": "Damaged",
+        "integer_ws0la67": "Affected",
```
Comment (ericboucher) [inline apps/frontend/translations/en.json:214]: These do not seem to appear in Flodd, should they be under Incident instead?

File: packages/interfaces/src/kobo/constants.ts [hunk 3]
```diff
@@ -30,6 +30,7 @@ export const IncidentMapping: MappingTypes = {
   drowning: '10',
   collapse: '11',
   weapon: '12',
+  shipwreck: '13',
```
Comment (ericboucher) [inline packages/interfaces/src/kobo/constants.ts:33]: I also saw a value 14 in the files you shared, should we add it too?

File: packages/interfaces/src/kobo/mapping/incident.ts [hunk 4]
```diff
@@ -132,6 +134,8 @@ export const incidentSpecificKeys = {
   [IncidentSpecific.CraftDam]: 'group_pk03i04/group_ub2hp14/CraftDam',
   [IncidentSpecific.CropAff]: 'group_gh6ag70/group_na4yi04/CropAff',
   [IncidentSpecific.CropDam]: 'group_gh6ag70/group_na4yi04/CropDam',
+  [IncidentSpecific.integer_ws0la67]: 'group_gh6ag70/group_az7bv13/CropAff',
```
Comment (ericboucher) [inline packages/interfaces/src/kobo/mapping/incident.ts:137]: The values on the right should be the corresponding keys from the forms

File: packages/interfaces/src/kobo/constants.ts (hunk 3 above)
Comment (ericboucher) [reply packages/interfaces/src/kobo/constants.ts:33]: Looks like it's also shipwreck (based on google trad) so I'm a bit confused

**Pull request merged**